TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/DriveConstants.java

```diff
@@ -33,7 +33,7 @@
      * MOTOR_VELO_PID with the tuned coefficients from DriveVelocityPIDTuner.
      */
     public static final boolean RUN_USING_ENCODER = true;
-    public static PIDCoefficients MOTOR_VELO_PID = new PIDCoefficients(0, 0, 0);
+    public static PIDFCoefficients MOTOR_VELO_PID = new PIDFCoefficients(0, 0, 0, getMotorVelocityF());
```

Comment:
Since you're making other changes, I would rename `getMotorVelocityF()` to `getDefaultMotorVelocityF()`.